app/src/main/java/id/ac/itb/ditlog/monitorandperformance/ChooseIndicator.java

```diff
@@ -211,13 +211,14 @@ public void onCheckboxClicked(View view) {
 
     public class indicatorGetter extends AsyncTask<Void, Void, ArrayList<IndicatorEntity>>{
 
+        public String auth = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJhZXAiLCJyb2xlSWQiOjQyMiwiZXhwIjoxNTIyMzM2Mzg1fQ.3nai_tQNWLObap18t8YjZ-RrtisOhlPLq7kI_zDgy1Gq99VNdWTicQ5o-c8BPTh2ZPRxBOhIqumAaCc-8F9-2A";
         public static final String SERVER_URL = "159.65.131.168:8080";
```

Comment:
erase this, use `BuildConfig.WEBSERVICE_URL` instead to refer the webservice url